airbyte_cdk/test/standard_tests/models/scenario.py

```diff
@@ -83,3 +135,38 @@ def __str__(self) -> str:
             return f"'{self.config_path.name}' Test Scenario"
 
         return f"'{hash(self)}' Test Scenario"
+
+    def without_expected_outcome(self) -> ConnectorTestScenario:
+        """Return a copy of the scenario that does not expect failure or success.
+
+        This is useful when running multiple steps, to defer the expectations to a later step.
+        """
+        return ConnectorTestScenario(
+            **self.model_dump(exclude={"status"}),
+        )
```

Comment:
@dbgold17 - Per your feedback, and after further consideration, I've refactored this as a tri-state value, and renamed from `expect_exception` to `expected_outcome`. That makes this `without_()` method make more sense, and then we separately have bools for `with_expecting_failure()` and `with_expecting_success()`. Some additional refactoring was needed, since the same code was called in other places, but this implementation makes a lot more sense and can handle more test scenarios.